Automattic/wp-calypso
# Blaze: Implement mini-carousel in /stats

#### Proposed Changes

| Mobile  | Desktop |
| :-------------: | :-------------: |
| <img src="https://user-images.githubusercontent.com/6586048/212279621-ed979854-137e-4b90-9a41-fce819551123.gif">  | <img src="https://user-images.githubusercontent.com/6586048/212279630-753b63d0-e641-4851-a50d-4f386bd16282.gif">|

Folow up issues: 

#### Testing Instructions



* Go to `/stats/day/:site?flags=stats-mini-carousel`
* Check if it works well on desktop and mobile
* Check if track click events are the same as before

#### Pre-merge Checklist





Fixes https://github.com/Automattic/dotcom-forge/issues/1460
